### ISSUE TITLE
Todo: LUKS encryption

### ISSUE THREAD

**Issue by noon92** [noon92/femtofox#23]

**Comment by joshbowyer** [comment]
The main consideration here is how to decrypt. Users would not be able to enter a password on boot so we would need to use a keyfile. Where would this be placed? For remote setups this would not be feasible, at least for full disk encryption. What may be best is to encrypt /home and put the keyfile in another partition, such as userdata (iirc this is /dev/mmcblk1p5), but then an attacker, if somehow able to get into the rootfs externally, could just see the location from fstab.

Basically, the disk is exotic enough to most people that encryption may not be necessary (env vs partition table), but encrypting anything on the disk will require the keyfile to also be physically present on the sdcard.
